financial-templates-lib/test/price-feed/CreatePriceFeed.js

```diff
@@ -148,6 +159,82 @@ contract("CreatePriceFeed.js", function(accounts) {
     );
   });
 
+  it("Default Uniswap Config", async function() {
+    // Given the collateral token is 0x1, the , it should always come first, meaning the config should always be inverted.
+    const collateralTokenAddress = "0x0000000000000000000000000000000000000001";
+
+    const constructorParams = {
+      expirationTimestamp: (Math.round(Date.now() / 1000) + 1000).toString(),
+      withdrawalLiveness: "1000",
+      collateralAddress: collateralTokenAddress,
+      finderAddress: Finder.address,
+      tokenFactoryAddress: TokenFactory.address,
+      priceFeedIdentifier: web3.utils.utf8ToHex("ETH/BTC"),
+      syntheticName: "Test UMA Token",
+      syntheticSymbol: "UMATEST",
+      liquidationLiveness: "1000",
+      collateralRequirement: { rawValue: toWei("1.5") },
+      disputeBondPct: { rawValue: toWei("0.1") },
+      sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
+      disputerDisputeRewardPct: { rawValue: toWei("0.1") },
+      minSponsorTokens: { rawValue: toWei("1") },
+      timerAddress: Timer.address
+    };
+
+    const emp = await ExpiringMultiParty.new(constructorParams);
+
+    const getIdBackup = web3.eth.net.getId;
+
+    // Modify web3 to say the chain id is mainnet temporarily.
+    web3.eth.net.getId = async () => 1;
+
+    const twapLength = 100;
+    const priceFeed = await createUniswapPriceFeedForEmp(logger, web3, networker, getTime, emp.address, { twapLength });
```

Comment:
So the advantage of this new implementation of `createUniswapPriceFeedForEmp` is that we can get programatically get the `pairAddress` and `inverted` properties?

Reply:
Correct.

Reply:
You can also get sensible defaults for the rest of the parameters, but yes, the important ones are the pair address and the inverted properties.